srtp_writer_future.go

```diff
@@ -90,7 +102,7 @@ func (s *srtpWriterFuture) Write(ctx context.Context, b []byte) (int, error) {
 		return value.(*srtp.WriteStreamSRTP).WriteContext(ctx, b)
 	}
```

Comment:
I forgot to change func name from `Write` to `WriteContext` above...

Reply:
@at-wat @OrlandoCo I can change that, but I am not sure what the answer is re: context yet?

* Is the performance good enough to do Context only?
* Are we providing two paths (`Write` and `WriteContext`), or removing context?

Reply:
Maybe better to ask @josharian as well

Reply:
:+1: we can wait for a response!

Personally how do you feel about rolling back the context changes and exposing `SetReadDeadline` and `SetWriteDeadline`? Are you really excited about adding Context, or do you consider it a 'nice to have'. 

@OrlandoCo and @jech are pretty performance sensitive. So I just want to find something that makes sense for everyone.

Reply:
I personally like to control it by context mainly for future maintainability and easiness of managing resources. Exposing SetDeadline can realize equivalent behavior, but bit complicated.
I don't have very urgent need for this.